app/controllers/base_drafts_controller.rb

```diff
@@ -19,6 +19,10 @@ def show
     end
 
     add_breadcrumb breadcrumb_name(get_resource.draft, resource_name), send("#{resource_name}_path", get_resource)
+
+    respond_to do |format|
```

Comment:
I think this is not working properly, have you tested both html and json versions with all draft types locally?
1. I think this is missing a `format.html`
2. The way this is called by the collection_drafts_controller is causing some issues that need to be resolved. If you want to discuss it at all, let me know.